src/app/components/media/MediaComponent.js

```diff
@@ -294,6 +294,9 @@ class MediaComponent extends Component {
     media.quote = media.media.quote;
     media.embed_path = media.media.embed_path;
 
+    // console.log(this.props.trololo.meajude.xocxoc); // eslint-disable-line no-console
+    console.log(JSON.parse('bli')); // eslint-disable-line no-console
```

Comment:
Stray console log!

Reply:
Thanks! Fixing this! Not only a console.log but one with a runtime error in it! :sweat_smile: